src/Workleap.DomainEventPropagation.Subscription.PullDelivery/EventPullerService.cs

```diff
@@ -69,6 +77,29 @@ private async Task StartReceivingEventsAsync(EventGridTopicSubscription eventGri
         }
     }
 
+    private async Task<EventProcessingStatus> HandleEventAsync(ICloudEventHandler cloudEventHandler, CloudEvent cloudEvent, CancellationToken stoppingToken)
+    {
+        try
+        {
+            await cloudEventHandler.HandleCloudEventAsync(cloudEvent, stoppingToken).ConfigureAwait(false);
+            return EventProcessingStatus.Handled;
+        }
+        catch (Exception ex)
+        {
+            switch (ex)
+            {
+                case EventDomainTypeNotRegisteredException:
+                case CloudEventSerializationException:
+                case EventDomainHandlerNotRegisteredException:
+                    this._logger.EventWillBeRejected(cloudEvent.Id, ex);
+                    return EventProcessingStatus.Rejected;
+                default:
+                    this._logger.EventWillBeReleased(cloudEvent.Id, ex);
```

Comment:
On voudrait peut-être rethrow ici? Le catch attrape toute exception possible et le log est seulement en level info. So on pourrait cacher des choses et les gens pourraient ne rien voir dans les logs as to why it's happening considérant qu'en prod les logs info sont probablement pas affichés par défaut

Reply:
J'ai fait un refactoring de la logique, je pense qu'il manquait un peu de precision dans les logs aussi et j'ai change les logs pour etre warning plutot